### PR TITLE
Use homePosition in LocationsService.getHomePosition

### PR DESCRIPTION
**Pull request by tbtz**
- abstract home position in endpoint
- use homeposition in LocationsService

MI-168

### CODE REVIEW

File: src/frontend/services/AuthService.js [hunk 1]
```diff
@@ -7,20 +7,20 @@ class AuthService {
...
-        config => {
-          if (this.isAuthenticated()) config.headers.authorization = JWTService.getJWT();
-          return config;
-        },
-        error => Promise.reject(error)
+      config => {
+        if (this.isAuthenticated()) config.headers.authorization = JWTService.getJWT();
+        return config;
+      },
+      error => Promise.reject(error)
```
Comment (beanduong) [inline src/frontend/services/AuthService.js]: ;

File: src/frontend/services/LocationService.js [hunk 2]
```diff
@@ -19,9 +20,9 @@ class LocationService {
 
   getHomePosition() {
     return new Promise(async (resolve, reject) => {
-      const userHasHomePosition = false; // Mocked
-      if (userHasHomePosition) {
-        // return home position
+      const homePosition = AuthService.getUser().homePosition
```
Comment (beanduong) [inline src/frontend/services/LocationService.js]: ;

Comment (tbtz) [reply src/frontend/services/LocationService.js]:
```suggestion
      const homePosition = AuthService.getUser().homePosition;
```

File: src/frontend/services/AuthService.js (hunk 1 above)
Comment (tbtz) [reply src/frontend/services/AuthService.js]:
```suggestion
      error => Promise.reject(error);
```

**Pull request merged**